scripts/netdep-graph-png.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
```

Comment:
he probably did, but 50% similarity breaks with small files pretty fast.

in this case, the usage lines were added in the middle of the file, so it no longer hit the 50% similarity metric.